src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/AatfDataToHomeViewModelMap.cs

```diff
@@ -21,9 +21,11 @@ public HomeViewModel Map(AatfDataToHomeViewModelMapTransfer source)
 
             var selectedAatfsOrAes = new List<AatfData>();
 
+            var filteredAatfList = RemoveOlderAatfs(source.AatfList);
+
             if (source.IsAE)
             {
-                foreach (var aatf in source.AatfList)
+                foreach (var aatf in filteredAatfList)
```

Comment:
Move the filtered list call before the If (source.IsAE) part as the same list is, probably dont need the foreach, could use linq query to filter based on facility type

Also need unit tests for the mapping changes